palindrome-products/palindrome-products_test.spec.js

```diff
@@ -7,7 +7,7 @@ describe("Palindrome", function() {
     var largest = palindromes.largest;
 
     expect(largest.value).toEqual(9);
-    expect([1, 9]).toEqual(largest.factors);
+    expect([[3, 3], [1, 9]], [[1, 9], [3, 3]]).toContain(largest.factors);
```

Comment:
In fact, in the JS track, the expected value has an additional array level. Instead of:

```
[ [3, 3], [1, 9] ], [ [1, 9], [3, 3] ]
```

that value in JS is:

```
[ [ [3, 3], [1, 9] ], [ [1, 9], [3, 3] ] ]
```

Running the JS exercise, `largest.factors` is `[ [ 1, 9 ], [ 3, 3 ] ]` while in this track `largest.factors` is `[1, 9]`. It seems the implemented solution has some errors.

I've checked JS and Ruby tracks, and they look the same. This track looks a little different. A good test could be to copy the JS code and paste it in this track, just to see if all tests pass.